src/logger.cpp

```diff
@@ -123,6 +126,11 @@ bool IsLoggingFunctionCalls()
     return miopen::IsEnabled(MIOPEN_ENABLE_LOGGING{}) && !IsLoggingDebugQuiet();
 }
 
+bool IsLoggingToRoctx()
+{
+    return miopen::IsEnabled(MIOPEN_ENABLE_LOGGING_ROCTX{}) && !IsLoggingDebugQuiet();
```

Comment:
@junliume [optimize] It is likely that `!IsLoggingDebugQuiet()` is not necessary here. Quieting is used during warm-up by the driver. The tracer records related to warm-up will not be annotated, which is not good. And finally, removal of this check would reduce overhead.

Reply:
Ping @mwootton for the updates